ruby_on_rails/app_initialisation.md

```diff
@@ -0,0 +1,86 @@
+# Initialise the Rails App
+
+* [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not. You can do this with `gem update rails`.
+
+* Start a new Rails project using the usual `rails new -d postgresql [project-name]` where the `project-name` is exactly the one you chose before.
```

Comment:
but then the initial test command doesn't work...I agree but I'd consider it a further improvement and create an issue for that.